docs: Middleware fetching is not cached (#80378)
vercel/next.js: docs/01-app/03-building-your-application/01-routing/14-middleware.mdx

- Slow data fetching
- Session management

> These fetch options, `options.cache`, `options.next.revalidate`, or `options.next.tags`, have no effect when used on Middleware.

Author: ```suggestion
Using fetch with `options.cache`, `options.next.revalidate`, or `options.next.tags`, has no effect in Middleware.
```
